examples/js/postprocessing/AdaptiveToneMappingPass.js

```diff
@@ -186,7 +186,7 @@ THREE.AdaptiveToneMappingPass.prototype = {
 			this.previousLuminanceRT.dispose();
 
 		}
-		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBFormat };
+		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBAFormat };
```

Comment:
Thanks @MiiBond. If you discover an answer, please post. The code does appear to work on other browsers.